src/client/src/index.tsx

```diff
@@ -11,9 +11,28 @@ const SimpleLauncher = lazy(() => import('./apps/SimpleLauncher'))
 
 const urlParams = new URLSearchParams(window.location.search)
 
+function getEnv() {
+  const serviceUrl = process.env.REACT_APP_BROKER_HOST || window.location.host;
+  if (serviceUrl.includes('localhost')) return 'localhost';
+  const envMatch = serviceUrl.match(/web-([a-zA-Z]+)\.adaptivecluster\.com/);
+  return envMatch ? envMatch[1] : undefined
+}
+
+const envTitles = {
+  localhost: '(LOCAL)',
+  dev: '(DEV)',
+  uat: '(UAT)',
+  demo: '',
+}
+
 async function init() {
   console.info('BUILD_VERSION: ', process.env.REACT_APP_BUILD_VERSION)
 
+  const env = getEnv()
+  if (env !== undefined) document.title = `${document.title} ${envTitles[env]}`
```

Comment:
What will happen if `env` is undefined, will the title just revert to "Reactive Trader Cloud"?

Reply:
yes, unless you wanted this to be set to something specific?

Reply:
I am actually unsure - should we set it to "Undefined" or "Unknown" to indicate the issue? Otherwise it is the same as for Demo 🤔

Reply:
The browser will now display (UNKNOWN) if the environment returns undefined in order to differentiate itself from Demo